site/.vitepress/theme/components/home/HeroDisplay.vue

```diff
@@ -18,8 +18,12 @@ import LinkButton from './LinkButton.vue';
                 <span class="from-blue-400 via-purple-700 to-blue-800 bg-gradient-to-r bg-clip-text text-transparent">Code</span>
             </h1>
             <h1 class="hidden dark:block text-4xl font-bold tracking-tight sm:text-7xl from-blue-200 via-purple-700 to-blue-600 bg-gradient-to-r bg-clip-text text-transparent animate-fade-up">More Performance, Less Code</h1>
-            <p class="mt-6 text-lg leading-8 text-black dark:text-gray-200">Lightweight, open-source framework for developing hyper fast animated applications</p>
-            <LinkButton class="mt-10" href="https://lightningjs.io/v3-docs/blits/intro.html" label="Beta"/>
+            <p class="mt-6 text-lg leading-8 text-black dark:text-gray-200">Lightweight, open-source framework for developing hyper fast animated applications. Get started with:</p>
+            <div class="flex items-center justify-center gap-10 py-5">
+                <a href="https://lightningjs.io/v3-docs/blits/intro.html" target="_blank" class="transition ease-out duration-300 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:-translate-y-1 hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"> Blits <span aria-hidden="true">→</span></a>
+                or
+                <a href="https://lightning-js.github.io/solid/intro.html" target="_blank" class="transition ease-out duration-300 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:-translate-y-1 hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"> SolidJS <span aria-hidden="true">→</span></a>
+            </div>
```

Comment:
We should think about the UX here, in the future React and maybe others will join. I think we shouldn't fragment the site with too much and keep the alternatives in 1 separate place.

Reply:
Will React + React Native be official projects? Are they part of Lightning? We should think about the UX, we have two projects and only one is being pointed to.